README.md

```diff
@@ -580,4 +600,7 @@ These are some example projects using CML.
 - [CML with DVC to pull data](https://github.com/iterative/cml_dvc_case)
 - [CML with Tensorboard](https://github.com/iterative/cml_tensorboard_case)
 - [CML with a small EC2 instance](https://github.com/iterative/cml-runner-base-case)
+  :key:
 - [CML with EC2 GPU](https://github.com/iterative/cml_cloud_case)
```

Comment:
need to check which of these really need a PAT

Reply:
Regarding that, #625 might give you some ideas.

Reply:
ent through this list, onlt the last 2 items needed a PAT